BLUEBUTTON-209 Add admin endpoints for count metrics

dtisza1 opened the pull request:
- Add User records count

- Add User active records count

- Add User staff records count

- Add User super-user/admin records count

- Add UserProfile records count

- Add DEV UserProfile records count

- Add BEN UserProfile records count

NOTE: Still working on a few more endpoints related to Jira BB #209. Wanted to make sure the approach is OK? It is testing OK so far!

Also I'm using the user.is_staff attribute for authentication, so that non-admin users can get access to the end points.

---Dave---

whytheplatypus commented on apps/metrics/views.py:
```diff
@@ -0,0 +1,62 @@
+@api_view(['GET'])
+@renderer_classes((JSONRenderer,))
+def metric_response(request, content):
+    if request.user.is_authenticated() and request.user.is_staff:
```
Enforce permissions with [permission classes](http://www.django-rest-framework.org/api-guide/permissions/)

whytheplatypus replied:
And possibly https://github.com/django/django/blob/master/django/contrib/admin/sites.py#L188-L210 as well in `urls.py`

whytheplatypus commented on apps/metrics/views.py:
```diff
@@ -0,0 +1,62 @@
+
+
+# User records count
+def total_user_count(request, format=None):
```
Put all of these queries together, or look into implementing [filters](http://www.django-rest-framework.org/api-guide/filtering/).

whytheplatypus replied:
filters could also be future work and this could just default to the `bene` count for now.

dtisza1 commented:
I rewrote this based on review comments and discussion with Gage. The following was changed:
* Changed to using Django class views.
* Changed to use rest_framework permissions.
* Changed to provide beneficiaries, applications, and tokens endpoint views that can be later expanded upon.

---Dave---

whytheplatypus commented on apps/metrics/views.py:
```diff
@@ -0,0 +1,70 @@
+
+    def get(self, request, format=None):
+        content = {
+            'beneficiary_count': UserProfile.objects.filter(user_type='BEN').count()
```
rename these keys to just `count` or `total`
